spec/services/attribute_asserter_spec.rb

```diff
@@ -61,25 +63,19 @@
         end
 
         context 'authn request does not specify bundle' do
-          it 'contains DEFAULT_BUNDLE' do
-            expect(user.asserted_attributes.keys).to match_array(
-              AttributeAsserter::DEFAULT_BUNDLE + [:uuid]
-            )
+          it 'only returns uuid' do
```

Comment:
Yes, we discussed this in Slack, and Amos said he thinks we should default to none. The SP has to specifically request attributes if it wants them.